.github/workflows/test_package.yml

```diff
@@ -0,0 +1,90 @@
+name: Build wheels TestPyPI
```

Comment:
I see the point, but am not 100% convinced it is a good idea to have all this duplicated in regards to package.yml.
Maybe we could extract all jobs except upload_to_pypi to seperate `build.yml` file that would produce artifacts, and then have `release.yml` and `test_release.yml` workflows that depend on build, and can be triggered using workflows dispatch, so that for each build you can first relee test and later to prod?

alternatively I suppose github allows for workflow approvals but I haven't used it so not sure how it works in action.

what do you think?

Reply:
I thought about creating a choice (pypi / test pypi) in workflow dispatch.

But I like the build / release separation idea, as we wouldn't have to wait for the wheels to build after releasing on test pypi. I'll think about it.

Reply:
For now I've gone with the choice option.

Reply:
Fine for now I guess, but I really think we should rather upload the same package than rebuild. Perhaps something to do in the future.